profile_library/ikea/LED2104R3/model.json

```
@@ -0,0 +1,15 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2025-12-22T22:20:22.704853",
+  "device_type": "light",
+  "measure_description": "Measured with utils/measure script",
```

```suggestion
  "measure_description": "Measured with utils/measure script. Used 40W incandescent bulb as dummy load.",
```

Let's add the note about dummy load in the model.json